client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -11,7 +10,7 @@ export const EDUCATION_EARN = 'home-education-earn';
 const EducationEarn = ( { siteSlug } ) => {
 	const translate = useTranslate();
 	const { isModalOpen, openModal, closeModal } = useRouteModal(
-		'coursePaymentsModal',
+		'myHomeCoursePaymentsModal',
```

Comment:
Can we keep using `coursePaymentsModal` in the off-change that someone already bookmarked the previous URL?

Reply:
Done